tests/onnx/test_low_opset_model.py

```diff
@@ -0,0 +1,125 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+# pylint: disable=redefined-outer-name
+
+import pytest
+
+import os
+
+import torch
+from torchvision import models
+import onnx
+
+from nncf.experimental.onnx.datasets.imagenet_dataset import infer_input_shape
+from tests.common.helpers import TEST_ROOT
+from tests.onnx.quantization.common import ModelToTest
+
+
+MODEL_NAMES = [
+    'resnet18',
+    'mobilenet_v2',
+    'mobilenet_v3_small',
+    'inception_v3',
+    'googlenet',
+    'vgg16',
+    'shufflenet_v2_x1_0',
+    'squeezenet1_0',
+    'densenet121',
+    'mnasnet0_5',
+]
+
+MODELS = [
+    models.resnet18(),
+    models.mobilenet_v2(),
+    models.mobilenet_v3_small(),
+    models.inception_v3(),
+    models.googlenet(),
+    models.vgg16(),
+    models.shufflenet_v2_x1_0(),
+    models.squeezenet1_0(),
+    models.densenet121(),
+    models.mnasnet0_5(),
+]
+
+INPUT_SHAPE = [1, 3, 224, 224]
+INPUT_KEY = 'input.1'
+LOW_OPSET_VERSIONS = [6, 7, 8, 9]
```

Comment:
If opset 6 is not supported, please remove it.

Reply:
Done